jsonpickle/util.py

```diff
@@ -577,9 +577,11 @@ def itemgetter(obj, getter=operator.itemgetter(0)):
     return compat.ustr(getter(obj))
 
 
-def items(obj):
+def items(obj, exclude_keys=[]):
```

Comment:
Let's make this `def items(obj, exclude=()):` -- Python trivia/best-practice is to avoid mutable contains as defaults for function keyword parameters.

The rationale for this family of suggestions is that keyword parameters are stored as global variables as part of the `def` object, and code that inadvertently mutates them ends up mutating a global variable that accumulates state. We don't mutate stuff here, but we can stick to the spirit of avoiding that construct by defaulting to an empty tuple instead of a list.

This is not an issue in practice in this code since we don't mutate, but if you use lists, sets or dicts as default values it can cause experienced python users to do a double-take while reading/reviewing the code.

It's funny.. `util.items()` was originally added for python2+3 compatibility and has a TODO that we should get rid of it. If we're going to start relying on it then we should replace the docstring below to not suggest its removal.

Reply:
Agree.

Reply:
Agree